### ISSUE TITLE
CCEaseOut is wrong

### ISSUE THREAD

**Issue by collinjackson** [cocos2d/cocos2d-objc#1164]
I saw this filed in the forums: http://forum.cocos2d-swift.org/t/cceaseout-is-wrong/11179

**Comment by slembcke** [comment]
@andykorth came across that recently in #1125. I don't remember the details off the top of my head. IIRC, they don't really work as designed, and that they are broken by design.

I'll attach this to 3.4 for now, but it seems likely that we'll need to settle for a hack in 3.4 and fix the API in v4 instead.

**Comment by collinjackson** [comment]
Sounds good, seems fine to fix in v4, just wanted to make sure it doesn't get lost.

**Comment by andykorth** [comment]
Yeah, I'd like to some degree of an overhaul of actions for v4. Other issues on my radar are https://github.com/cocos2d/cocos2d-swift/issues/530 and https://github.com/cocos2d/cocos2d-swift/issues/925. The latter one will probably be fixed as part of CCDirector fixes, which necessitate some changes to underlying scheduling and actions... 

CCActionEaseOut is one of several confusing actions, and using a rate of 0 or 1 will definitely not give you what you expect. So since these would all be API changes, v4 would be a good time to do them.
